xrpl_api/src/api/subscribe.rs

```diff
@@ -70,23 +67,23 @@ impl SubscribeRequest {
 
     /// When you subscribe to one or more order books with the books field, you
     /// get back any transactions that affect those order books.
-    pub fn books(books: &[Book]) -> Self {
+    pub fn books(books: Vec<Book>) -> Self {
```

Comment:
Why is this one better? 🤔

Reply:
Never mind, I think I understand the rationale.

Reply:
Yes, it is to let caller control the allocation (potentially cloning) instead of forcing cloning.